feat(photoprism) **BREAKING CHANGES** storage changes

kqmaverick opened the pull request:
**Description**
Use storage settings for photoprism originals and import.

⚒️ Fixes  # 

**⚙️ Type of change**

- [X] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [X] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [X] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

kqmaverick committed:
storage changes

stavros-k commented on charts/stable/photoprism/values.yaml:
```diff
@@ -16,6 +16,12 @@ persistence:
+    mountPath: "/originals"
+  import:
+    enabled: true
+    mountPath: "/import"
```
```suggestion
  originals:
    enabled: true
    mountPath: "/photoprism/originals"
  import:
    enabled: true
    mountPath: "/photoprism/import"
```

Maybe keep those as they were?
That way people would just need to remove the additional storage and just use the "built in" gui to point to their data

stavros-k reviewed (changes requested):
see comment

kqmaverick committed:
Update charts/stable/photoprism/values.yaml

Was worried it would cause the same issues as when we messed with plex plug-ins since /photoprism is already mapped. Guess we see if it passes checks.

Co-authored-by: Stavros Kois <47820033+stavros-k@users.noreply.github.com>
Signed-off-by: kqmaverick <121722567+kqmaverick@users.noreply.github.com>

kqmaverick committed:
let me try this before revert

I noticed the ENVs were not changed. Lets just test this before I revert.

Signed-off-by: kqmaverick <121722567+kqmaverick@users.noreply.github.com>

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
